src/include/catalog/catalog_defs.h

```diff
@@ -2,6 +2,22 @@
 #include "common/strong_typedef.h"
 
 namespace terrier::catalog {
+
```

Comment:
Would be helpful to add a comment, refering the reader to catalog.h, where the numbering of oids is explained.

Reply:
Added.